db/migrate/20170107025014_create_order_items.rb

```diff
@@ -0,0 +1,9 @@
+class CreateOrderItems < ActiveRecord::Migration[5.0]
+  def change
+    create_table :order_items do |t|
```

Comment:
@drod1000 It would be painful, but as @Dpalazzari already pointed out to keep consistency with the rest of the code, `order_outings` would be a better naming here. FYI @njgheorghita